src/app/search/autocomplete.ts

```diff
@@ -145,7 +145,9 @@ export function filterSortRecentSearches(query: string, recentSearches: Search[]
   //       we aren't going to do different sorts for query vs. non-query
   const recentSearchesForQuery = query
     ? recentSearches.filter((s) => s.query.includes(query))
-    : Array.from(recentSearches);
+    : recentSearches
+    ? Array.from(recentSearches)
+    : [];
```

Comment:
https://sentry.io/share/issue/fae80d19aa9f437793d8208932d5e079/

Reply:
This should always be defined now as well